src/codeu/chat/UserMap.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import codeu.chat.client.simplegui.NewAccount;
+
+/**
+ * Created by Suveena on 5/24/17.
+ */
+
+/* Basic structure that holds all the users. Create this map upon starting GUI.
+Each time we add a new user, check to make sure that they don't already exist.
+If they do not, add them to the map.*/
```

Comment:
this is what code review is meant to catch ;) hopefully still early enough for re-think ^__^